xds/src/test/java/io/grpc/xds/RingHashLoadBalancerTest.java

```diff
@@ -1096,10 +1096,11 @@ public void duplicateAddresses() {
     assertThat(result.getStatus().isOk()).isFalse();  // fail the RPC
     assertThat(result.getStatus().getCode())
         .isEqualTo(Code.UNAVAILABLE);  // with error status for the original server hit by hash
-    assertThat(result.getStatus().getDescription()).isEqualTo(
-        "Ring hash lb error: EDS resolution was successful, but there were duplicate "
-            + "addresses: Address: FakeSocketAddress-server1, count: 2; "
-            + "Address: FakeSocketAddress-server2, count: 3");
+    String description = result.getStatus().getDescription();
+    assertThat(description).startsWith(
```

Comment:
This isn't really needed.  If someone decides to change the text in the future that is fine.

Reply:
We're always on the fence whether error messages should be tested. Anyway, I've noticed it after merging :)